timescaledb/how-to-guides/migrate-data/troubleshooting.md

```diff
@@ -0,0 +1,13 @@
+# Troubleshooting data migration problems
+
```

Comment:
Can we add the boilerplate Troubleshooting stuff in here? You can see it in the caggs section.